dev/devicelab/lib/command/upload_results.dart

```diff
@@ -38,6 +39,15 @@ class UploadResultsCommand extends Command<void> {
     final String? builderName = argResults!['luci-builder'] as String?;
     final String? testStatus = argResults!['test-status'] as String?;
 
+    // Upload metrics to metrics_center from test runner.
+    // The upload step will be skipped from cocoon once this is validated.
+    try {
+      await uploadToMetricsCenter(resultsPath);
+      print('Successfully uploaded metrics to metrics center');
+    } on Exception catch (e, stacktrace) {
+      print('Uploading metrics failure: $e\n\n$stacktrace');
```

Comment:
We shouldn't try/catch this. Otherwise it'll silently fail to report results

Reply:
This is intensional here to unblock cocoon updates in case of errors before we validate and remove the cocoon logic.

Reply:
A TODO comment would work well

Reply:
Added. Here is the TODO bug: https://github.com/flutter/flutter/issues/88484